node-template/runtime/src/lib.rs

```diff
@@ -261,13 +263,19 @@ impl_runtime_apis! {
 		}
 	}
 
-	impl consensus_aura::AuraApi<Block, AuraId> for Runtime {
+	impl consensus_aura::AuraApi<Block, AuraId, ed25519::Signature> for Runtime {
 		fn slot_duration() -> u64 {
 			Aura::slot_duration()
 		}
 		fn authorities() -> Vec<AuraId> {
 			Aura::authorities()
 		}
+
+		fn construct_equivocation_report_call(
+			_proof: AuraEquivocationProof<<Block as BlockT>::Header,ed25519::Signature>
+		) -> Vec<u8> {
+			vec![]
```

Comment:
This, however, should not be empty.  It seems to me that a proof is definitely required.  Even if it is not, there should be documentation explaining why the empty vector is a valid choice.